### PR TITLE
add some guidance on lang attributes

### PR DESCRIPTION
**Pull request by hollsk**
:goose:

https://www.scientificamerican.com/article/the-standard-breeds-of-geese/

### CODE REVIEW

**Comment by jpw** [comment]
I enjoyed this PR very much.

File: accessibility/common-remediations.md [hunk 1]
```diff
@@ -416,6 +419,85 @@ We _don’t_ do this:
...
+
+The language specified in the HTML `lang` attribute doesn't match the language used for the main page. 
+
+#### Who's affected by the problem?
+
+* Screen reader users
+
+#### Why's it a problem?
+
+Words are pronounced differently in different languages. Screen reader software is designed to respect these pronouncation differences when it encounters web pages that specify their main language.
```
Comment (josebolos) [inline accessibility/common-remediations.md]: pronouncation?

Comment (hollsk) [reply accessibility/common-remediations.md]: oops, yes. I always misspell that

```suggestion
Words are pronounced differently in different languages. Screen reader software is designed to respect these pronunciation differences when it encounters web pages that specify their main language. 
```

Comment (hollsk) [reply accessibility/common-remediations.md]: oh i merged already. will put it in another PR

**Pull request merged**